Calculator.cs

```diff
@@ -0,0 +1,30 @@
+   //using CalcClassLibrary;
+
+namespace QAAutomation
+{
+
+    public class Calculator
+    {
+         public static void Main(string[] args)
+
+
+         {
+            
+            Console.WriteLine("Enter two numbers ");
+            int A = Convert.ToInt32(Console.ReadLine());
+            int B = Convert.ToInt32(Console.ReadLine());
+
+            Algebra alg1 = new Algebra();
+
```

Comment:
What does this code do if there is no constructor? As well, we specify the variables as var, this is a common practice.